core/coreunix/test/metadata_test.go

```diff
@@ -153,6 +153,7 @@ func addFileToBtfs(node *core.IpfsNode, data []byte, metadata string) (ipath.Pat
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
 	defer cancel()
 
+	adder.AddInProcess = true
```

Comment:
This is weird - why would a test force change this variable to true? This variable should be set at the beginning of the actual processing code.